modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -103,6 +107,10 @@ interface ObjectRelationship {
 	type: ObjectRelationshipType;
 }
 
+type NotificationTemplateType = 'email' | 'userNotification';
```

Comment:
@igor-franca Those two were correct before. We're following the order of first declaring interfaces and later types, but we also need to respect the precedence of declaration and usage.